q2_gneiss/composition/_method.py

```diff
@@ -48,6 +50,51 @@ def ilr_phylogenetic_differential(
     return diff_balances, t
 
 
+def ilr_phylogenetic_posterior_differential(
+        posterior: xr.Dataset, tree: skbio.TreeNode,
+        minimax_filter: bool = True
+) -> (xr.DataArray, pd.DataFrame, skbio.TreeNode):
+    dataset, trimmed_tree = _xarray_match_tips(posterior, tree, 'features')
+    trimmed_tree = rename_clades(trimmed_tree)
+    # TODO: watch out for this indexing with diff. The ordering of the dims
+    # could mess things up here.
+    posterior_data = np.array(dataset['diff'].values)
```

Comment:
we should replace this with a pandas pivot so that the dimension order doesn't matter.